app/views/splash/index.html.erb

```diff
@@ -1,15 +1,15 @@
-<h1>Splash</h1>
-
 <div class="logo">
-    aqui va un logo
+    <%= image_tag "logo.png", alt: 'Logo' %>
 </div>
 
-<div class="signin">
-    <%= link_to "Sign in", new_user_registration_path %>
-</div>
 <div class="login">
-    <%= link_to "Log in", new_user_session_path %>
+    <%= link_to "LOG IN", new_user_session_path %>
 </div>
 
+<div class="signup">
+    <%= link_to "SIGN UP", new_user_registration_path %>
+</div>
+
+
 
 
```

Comment:
@Dachrono EXCELLENT WORK!!! I love your Splash screen, the design is very original and the name of the app looks wonderful! Congratulations

Reply:
I really appreciate yours comments